src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -765,23 +751,15 @@ private static string RetrieveProcessUserName(Process process)
                 }
 
                 var tokenUser = Marshal.PtrToStructure<Win32Native.TOKEN_USER>(tokenUserInfo);
-
-                // Max username is defined as UNLEN = 256 in lmcons.h
-                // Max domainname is defined as DNLEN = CNLEN = 15 in lmcons.h
-                // The buffer length must be +1, last position is for a null string terminator.
-                int userNameLength = 257;
-                int domainNameLength = 16;
-                Span<char> userNameStr = stackalloc char[userNameLength];
-                Span<char> domainNameStr = stackalloc char[domainNameLength];
-                Win32Native.SID_NAME_USE accountType;
-
-                // userNameLength and domainNameLength will be set to actual lengths.
-                if (!Win32Native.LookupAccountSid(null, tokenUser.User.Sid, userNameStr, ref userNameLength, domainNameStr, ref domainNameLength, out accountType))
```

Comment:
Is `Win32Native.LookupAccountSid` used elsewhere? If not, then better clean it up too.

Reply:
It is but I don't think it would be good to change that in this same PR.

Reply:
I'm fine that you submit a separate PR for the cleanup. I'm just worried that we will forget about it once merging this PR :)

Reply:
Let's leave it for another PR, keep this one self contained and it will allow us to review each usage individually to see if it can be swapped over.